### PR TITLE
Update kernel sources to 5.4.91

### PR DESCRIPTION
**Pull request by christopherco**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update kernel source to 5.4.91. New kernel source contains fixes for many kernel CVEs flagged by our tooling so address the CVEs.

###### Change Log  

- Update kernel-headers, kernel, kernel-hyperv, and hyperv-daemons specs to use 5.4.91
- Remove patch that fixes framebuffer mapping in HyperV. Integrated in 5.4.91
- Refresh version numbers for kernel-signed- specs
- Update toolchain to use 5.4.91 source when building kernel headers
- Mark the following CVEs as fixed: CVE-2020-29569, CVE-2020-28374, CVE-2020-36158
- Update cgmanifest's download URLs to point to 5.4.91 source location
- Removed CONFIG_PGTABLE_MAPPING from ARM64 config. This config is no longer supported in 5.4.91.

###### Does this affect the toolchain?  

**YES**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2020-29569
- https://nvd.nist.gov/vuln/detail/CVE-2020-28374
- https://nvd.nist.gov/vuln/detail/CVE-2020-36158

###### Test Methodology

- Local x86_64 build, pipeline ARM64 build

### CODE REVIEW

File: SPECS/kernel/kernel.spec [hunk 1]
```diff
@@ -15,7 +15,6 @@ Source2:        config_aarch64
 # Arm64 HyperV support required patch
 Patch0:         ver5_4_72_arm64_hyperv_support.patch
 Patch1:         efi-libstub-tpm-enable-tpm-eventlog-function-for-ARM.patch
-Patch2:         fix-gui-installer-mmap.patch
```
Comment (PawelWMS) [inline SPECS/kernel/kernel.spec:18]: Have you also removed the file itself? I cannot find it in this PR.

Comment (christopherco) [reply SPECS/kernel/kernel.spec:18]: Good catch. The patch file is still present in the directory. Will remove.

**Review by PawelWMS — APPROVED** [review]

**Review by rlmenge — APPROVED** [review]

**Review by jslobodzian — APPROVED** [review]
:shipit:

File: SPECS/kernel/config_aarch64 [hunk 2]
```diff
@@ -1007,7 +1007,6 @@ CONFIG_ZPOOL=y
 CONFIG_ZBUD=y
 CONFIG_Z3FOLD=m
 CONFIG_ZSMALLOC=y
-CONFIG_PGTABLE_MAPPING=y
```
Comment (PawelWMS) [inline SPECS/kernel/config_aarch64:1010]: Maybe also mention that in the changelog to make it easier to track down.

**Review by PawelWMS — APPROVED** [review]

File: SPECS/kernel/config_aarch64 (hunk 2 above)
Comment (christopherco) [reply SPECS/kernel/config_aarch64:1010]: Updated PR description. Not sure if it is needed in the changelog entry itself.

**Pull request merged**